rpaas/nginx/configuration_render.go

```diff
@@ -190,6 +190,18 @@ http {
 
     {{template "http" .}}
 
+		server {
+			listen 8800;
+
+{{if .Config.VTSEnabled}}
+			location /status {
+				vhost_traffic_status_display;
+        vhost_traffic_status_display_format prometheus;
```

Comment:
Missing some white spaces here? Btw, make sure to use white spaces instead of tabs.

Reply:
Indeed. I fixed it, thanks!